great-idea/services.html

```diff
@@ -4,103 +4,120 @@
+  eleifend. Mauris euismod facilisis iaculis.</p>
+        <button>Learn More</button>
+      </div>
 
-Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
-eleifend. Mauris euismod facilisis iaculis.
 
-Learn More
+      <div class="Web-dev-div">
+        <h4>Web Development</h4>
 
+        <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+  eleifend. Mauris euismod facilisis iaculis.</p>
+        <button>Learn More</button>
 
+      </div>
 
-UX / UI
 
-Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
-eleifend. Mauris euismod facilisis iaculis.
+      <div class="ios-div">
+        <h4>iOS Development</h4>
 
-Learn More
+        <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+  eleifend. Mauris euismod facilisis iaculis.</p>
+        <button>Learn More</button>
+      </div>
 
+      <div class="android-div">
+        <h4>Android Development</h4>
 
+        <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+  eleifend. Mauris euismod facilisis iaculis.</p>
+        <button>Learn More</button>
+      </div>
+</section>
 
-Digital Marketing
 
-Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
-eleifend. Mauris euismod facilisis iaculis.
+<div class="bottom-content">
+  <div class="text-content">
+<h4>Some Facts About Our Services</h4>
 
-Learn More
+<p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+eleifend. Mauris euismod facilisis iaculis.</p><br>
```

Comment:
Instead of a <br> you could also have targeted your ul in CSS and used some margin to add white space between the paragraph and list.